Makefile

```diff
@@ -3,11 +3,12 @@ OPAM_LIBDIR=$(DESTDIR)$(shell opam config var lib)
 
 .PHONY: build release install uninstall clean test doc reindent
 
+release:
```

Comment:
You may want to specify this as the DEFAULT_GOAL.

Reply:
The first target in a Makefile is the default.

Reply:
Yes, I'm only suggesting it to make it more explicit and less prone to breakage if someone decides to go and reorder the targets - but, ok, I don't have a strong opinion on this.